data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/parser/LogstashVisitor.java

```diff
@@ -0,0 +1,171 @@
+package org.opensearch.dataprepper.logstash.parser;
+
+import org.opensearch.dataprepper.logstash.exception.LogstashParsingException;
+import org.opensearch.dataprepper.logstash.LogstashBaseVisitor;
+import org.opensearch.dataprepper.logstash.LogstashParser;
+import org.opensearch.dataprepper.logstash.model.LogstashConfiguration;
+import org.opensearch.dataprepper.logstash.model.LogstashPlugin;
+import org.opensearch.dataprepper.logstash.model.LogstashAttribute;
+import org.opensearch.dataprepper.logstash.model.LogstashAttributeValue;
+import org.opensearch.dataprepper.logstash.model.LogstashPluginType;
+import org.opensearch.dataprepper.logstash.model.LogstashValueType;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.LinkedList;
+import java.util.ArrayList;
+
+/**
+ * Class to populate Logstash configuration model POJO's using ANTLR
+ *
+ * @since 1.2
+ */
+
+@SuppressWarnings("rawtypes")
+public class LogstashVisitor extends LogstashBaseVisitor {
+
+    private List<LogstashPlugin> logstashPluginList;
+    private final Map<LogstashPluginType, List<LogstashPlugin>> pluginSections = new LinkedHashMap<>();
+    private final Map<String, Object> hashEntries = new LinkedHashMap<>();
+
+    @Override
+    public Object visitConfig(LogstashParser.ConfigContext ctx) {
```

Comment:
can we rename this to `LogstashParser.ConfigContext configContext`?

Reply:
Yeah, I can rename the parameters.